apps/mark-scan/backend/src/custom-paper-handler/state_machine.test.ts

```diff
@@ -140,14 +140,19 @@ function expectMockPaperHandlerStatus(
   expect(mockDriver.getMockStatus()).toEqual(mockStatus);
 }
 
-beforeAll(async () => {
-  ballotPdfData = await renderBmdBallotFixture({
-    electionDefinition: electionGeneralDefinition,
-    frontPageOnly: true,
-  });
-  scannedBallotFixtureFilepaths =
-    await writeFirstBallotPageToImageFile(ballotPdfData);
-});
+beforeAll(
+  async () => {
+    ballotPdfData = await renderBmdBallotFixture({
+      electionDefinition: electionGeneralDefinition,
+      frontPageOnly: true,
+    });
+    scannedBallotFixtureFilepaths =
+      await writeFirstBallotPageToImageFile(ballotPdfData);
+  },
+  // Increase timeout for this hook only because ballot fixture
+  // rendering can take a few seconds
+  10_000
+);
 
 beforeEach(async () => {
```

Comment:
I've also seen the `beforeEach` fail with a timeout error as well. Maybe we bump the timeout for all of the hooks?